jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -595,6 +595,26 @@
                 # Increase delete collection parallelism
                 export TEST_CLUSTER_DELETE_COLLECTION_WORKERS="--delete-collection-workers=16"
                 export KUBE_NODE_OS_DISTRIBUTION="debian"
+        - 'gce-federation-1.4':  # kubernetes-e2e-gce-federation-1.4
+            # This test depends on the artifacts generated by the trigger-job. Be careful while changing this test.
+            trigger-job: 'kubernetes-federation-build-1.4'
+            description: 'Run all federation e2e tests.'
+            timeout: 300
+            emails: 'quinton@google.com, colin.hom@coreos.com, madhusudancs@google.com'
+            test-owner: 'madhusudancs'
+            job-env: |
+                export PROJECT="k8s-jkns-e2e-gce-f8n-1-4"
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Federation\]"
+                export GINKGO_PARALLEL="n" # We don't have namespaces yet in federation apiserver, so we need to serialize
+                export FEDERATION="true"
+                export DNS_ZONE_NAME="k8s-federation-1-4.com."
+                export FEDERATIONS_DOMAIN_MAP="federation=k8s-federation-1-4.com."
+                export E2E_ZONES="us-central1-a us-central1-b us-central1-f" # Where the clusters will be created. Federation components are now deployed to the last one.
+                export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
+                export KUBE_GCE_ZONE="us-central1-f" #TODO(colhom): This should be generalized out to plural case
+                export KUBE_GCS_RELEASE_BUCKET=kubernetes-federation-release
+                export KUBE_GCS_DEV_RELEASE_BUCKET=kubernetes-federation-release
+                export KUBE_NODE_OS_DISTRIBUTION="debian"
```

Comment:
Is this required?
We should be fine not overwriting this and getting the default value?